tests/admin_views/test_skip_link_to_content.py

```diff
@@ -0,0 +1,144 @@
+from selenium.webdriver.common.keys import Keys
```

Comment:
We cannot import from `selenium` at the module level as it's an optional requirement.

Reply:
Thank you!!

Fixed.